src/_locales/en/messages.json

```diff
@@ -0,0 +1,38 @@
+{
+    "extensionDescription": {
+        "message": "Facebook Container isolates your Facebook activity from the rest of your web activity in order to prevent Facebook from tracking you outside of the Facebook website via third party cookies.",
+        "description": "Description of the extension."
+    },
+    "onFacebookHeader": {
+        "message": "$URL$ is in the Facebook Container.",
+        "description": "This is shown at the top of the panel pop-up when the user is on a Facebook website, which has been automatically opened into Facebook Container.",
+        "placeholders": {
+            "url": {
+                "content": "$1",
+                "example": "www.facebook.com"
+            }
+        }
+    },
+    "thisSite": {
+        "message": "This site",
+        "description": "This is shown as the $URL$ in the onFacebookHeader message if the user is on a page with an empty URL. E.g., about:home, about:addons, etc."
+    },
```

Comment:
I'm not sure this will work @flodolo? This enforces an order of putting "This site" before "is in the Facebook Container" which I'm sure doesn't work in all languages. 😢 But I'm not sure how else to express this with this particular code and messages syntax.

Reply:
Sorry, I have no clue what this does (that's the first webextension I look at). I need some pointer, ideally a screenshot.

Reply:
@groovecoder Maybe an explicit additional "This site is not in the Facebook Container." message string instead of replacing $URL$ with "This site" would work?

Reply:
Good idea @stoically !

Reply:
@flodolo okay, I did what @stoically suggested and removed the confusing `thisSite` message. The messages should all be straight-forward translations now. (?) Does it look good?

Reply:
Yes, makes sense